frontend/e2e/fixtures/create-new/config-step.ts

```diff
@@ -54,3 +54,19 @@ export const configWithoutBlockColumn = {
     token: process.env.E2E_TOKEN_JIRA as string,
   },
 };
+export const chartUnHappyPathConfig = {
+  unSelectBranch: 'main',
+  addNewBranch: ['ADM-669'],
```

Comment:
Are this `unSelectBranch` and `addNewBranch` the fields saved as a JSON?
the fixture config file exports should keep the principle - the data structure is from saved json.

Reply:
this is step data to update branch, don't need to save, and now I have create new data file to save constant